### PR TITLE
removed todos, added delete jars from basedir to build as part of clean

### PR DESCRIPTION
**Pull request by ayeshare**
supports #4418

### CODE REVIEW

**Comment by IanMayo** [comment]
@ayeshare - was this PR to resolve a specific issue?

**Comment by IanMayo** [comment]
@ayeshare - we have an Ant build problem in this PR:
````
he following error occurred while executing this line:
[ERROR] /home/travis/build/debrief/debrief/org.mwc.debrief.legacy/build.xml:134: The following error occurred while executing this line:
[ERROR] /home/travis/build/debrief/debrief/org.mwc.debrief.legacy/build.xml:84: The following error occurred while executing this line:
[ERROR] /home/travis/build/debrief/debrief/org.mwc.debrief.legacy/build.xml:53: Compile failed; see the compiler error output for details.
[ERROR] around Ant part ...<ant antfile="build.xml" dir="/home/travis/build/debrief/debrief/org.mwc.debrief.legacy" target="dist"/>... @ 4:107 in /home/travis/build/debrief/debrief/org.mwc.debrief.legacy/target/antrun/build-main.xml
[ERROR] -> [Help 1]
[ERROR] Failed to execute goal org.eclipse.rcptt:rcptt-maven-plugin:2.3.0:resources (default-resources) on project org.mwc.debrief.ui_test: The explicit location /home/travis/build/debrief/debrief/org.mwc.debrief.ui_test/../org.mwc.debrief.product/target/products/DebriefNG-linux.gtk.x86_64.zip is not valid -> [Help 2]

````

**Comment by IanMayo** [comment]
Ok, @ayeshare - clearly we need another fix for the missing `cmap_legacy.jar`

**Comment by ayeshare** [comment]
> Ok, @ayeshare - clearly we need another fix for the missing `cmap_legacy.jar`

Yes, I am working on it.

File: org.mwc.debrief.lite/build.xml [hunk 1]
```diff
@@ -151,7 +151,7 @@
 			<fileset dir="../org.mwc.debrief.legacy/libs" includes="*.jar"/>
 			<fileset dir="../org.mwc.cmap.jfreechart/libs" includes="*.jar"/>
 			<fileset dir="../org.mwc.cmap.geotools/libs" includes="*.jar"/>
-			<fileset dir="${basedir}" includes="cmap_legacy.jar"/>
+			<fileset dir="../org.mwc.debrief.legacy" includes="cmap_legacy.jar"/>
```
Comment (IanMayo) [inline org.mwc.debrief.lite/build.xml]: @ayeshare - wouldn't `org.mwc.cmap.legacy` be a tidier source folder to take it from?  That's where the plugin is created/

Comment (ayeshare) [reply org.mwc.debrief.lite/build.xml]: yes, but we are building cmap from debrief.legacy, so the basedir will be debrief.legacy.

**Comment by IanMayo** [comment]
Verified, dist now contains `cmap_legacy.jar`

**Pull request merged**